### PR TITLE
Lab snippet lore update

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Content "Updates lore in lab terminals"```

#### Purpose of change
Flyers, newspapers, and lab terminal snippets are getting out of date to the lore in game currently. For example, flyers are largely devoted to advertising CBMs that are rare at the consumer level, if they're available at all.

#### Describe the solution
Goes through and cleans up, updates, and expands lab terminals.

While I'm there, I also moved radio chatter to a new file. No changes were made to content or layout at this time.

#### Describe alternatives you've considered
Initially I split the lore addition by types of lab they come from. This will require some infrastructure changes to make it work as I want though, so I saved that work but reverted to the original format.

#### Testing
wip

### CODE REVIEW

**Comment by RafeHaab** [comment]
Can I help?

**Comment by I-am-Erk** [comment]
Thanks for the offer but it's not really a two person job right now

File: data/json/snippets/radio.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "First Fleet, this is Mount Wheater, we've send a helicopter with a squad to the location of the Guard Outpost.  If any survivors of the crash are to reach the outpost, contact the base, we are short on men and one of the big ones has just breached the perimeter.  Over.",
+      "USS Eagle of Freedom to all callsigns.  Operation Ocean 11 is in effect.  Repeat.  Operation Ocean 11 is in effect.  Sir, this should effectively reach about 150 units, but there is no response from literally anyone in hours.  Their coms winked out like candles in the wind.  And how do we even secure the port to load anyone in?",
+      "Seagull 54 to USS Eagle of Freedom.  It's as we predicted.  Going down on last reserves of fuel, with dropping evacuated unit out of question at that point.  This is our last call.  Will try emergency landing at …kshssssssk… Don't send us a rescue team, we know the odds.  Will be moving east towards the shore, and if we're not there in T minus 3 days, count us dead.  Proud to serve the country.  Over and out."
+    ]
+  }
+]
```
Comment (ZhilkinSerg) [inline data/json/snippets/radio.json]:
```suggestion
]

```

File: data/json/snippets/lab.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      "Dr. Takatoshi sent us interesting news: her lab serendipitously discovered that sufficient rapid teleports in a short span of time can strip out XE037 somehow, without harming the subject.  This has the unfortunate issues of needing an absolutely insane amount of electricity, and drawing the attention of what Dr. Takatoshi calls \"subplanar creatures\" and the rest of us call \"horrifying alien monstrosities\".  Still, it's a start.",
+      "Dr. Sidhu figured out a way to track what happens to XE037 when it's stripped out during teleportation, using a high-affinity biotracer and a probe that follows immediately behind the test subject.  XE037 migrates out of the body in small but significant quantities with every teleport.  We're attributing way too much intelligence to it here, but we can't help but feel this is some kind of intentional 'stowaway' effect, like it's shedding off the body to try to colonize a new dimension.  At the meeting, Dr. Sidhu told us to stop anthropomorphizing it.  He thinks it's no more a sign of intelligence than a cold virus coming out with a sneeze."
+    ]
+  }
+]
+
```
Comment (ZhilkinSerg) [inline data/json/snippets/lab.json]:
```suggestion
```

File: data/json/snippets/lab.json [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      "Herrera's group retained copies of the source General Carlsberg asked us to look into; with several of our facilities no longer responding, or other teams essentially 'going down with their ships', they've done their best to adapt the code to operate without Melchior's guidance.  It's likely several revisions old now, and mostly shoots at anything humanoid shaped with a heat signature, but our hope is it'll buy us all some time.",
+      "Following S37ZBE's fallout, IT has noted far too many attempts have been made to access other teams' drives and terminals, including certain unofficial directories.  Lack of sufficient monitoring means we have been unable to pinpoint the bad actors, but IT has proposed automatically dispatching the UGV's as a deterrant.",
```
Comment (tenmillimaster) [inline data/json/snippets/lab.json]:
```suggestion
      "Following S37ZBE's fallout, IT has noted far too many attempts have been made to access other teams' drives and terminals, including certain unofficial directories.  Lack of sufficient monitoring means we have been unable to pinpoint the bad actors, but IT has proposed automatically dispatching the UGV's as a deterrent.",
```

File: data/json/snippets/lab.json [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      "I swear they are talking to me.  No one believes me.  No one understands.  They whisper to me, at night, in the dark.  Please, please, make them stop.",
+      "Today we found a solution to the contamination problem, quite by accident.  It was found that XE037 is completely removed from a revivified body following a series of extremely-high energy 4th dimensional transpositions.  Unfortunately, this invariably allows subplane life forms to pass into the prime plan.  The cure is worse than the disease.  Nonetheless, we'll be informing XEDRA-40 via Melchior.  Perhaps Dr. Sidhu can make some use of this information.",
+      "oday we applied a very small sample of XE037, suspended in water, to subject TP92, prior to necessary termination.  During the postmortem examination, the subject went through a revivification progress.  Dr. Thoreaux was surprised, to say the least.  She bludgeoned it back to death with a microscope.",
```
Comment (tenmillimaster) [inline data/json/snippets/lab.json]:
```suggestion
      "Today we applied a very small sample of XE037, suspended in water, to subject TP92, prior to necessary termination.  During the postmortem examination, the subject went through a revivification progress.  Dr. Thoreaux was surprised, to say the least.  She bludgeoned it back to death with a microscope.",
```

File: data/json/snippets/lab.json [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+      "An experimental subject was accidentally terminated today, and revivification was again seen as in Dr. Thoreaux's report.  This time we were ready for it, and the specimen was kept for observation.  It displays no human intelligence, a true animated corpse.  We're all calling it a zombie, because seriously.  This is so cool.",
+      "Testing XE037 on mouse and cat subjects does not appear to display the same revivifying effect, oddly enough.  Acquiring non-human subjects, sadly, is a long and costly process, and research into this area is lacking.  We'll send for some monkeys.",
+      "Earlier conjecture that revivification occurred only in humanss was premature.  Per earlier experiments, exposure introduces a persistent, low level infection in all mammal subjects, but quickly enters a form of stasis and seems to go dormant.  In most of our experimental subjects this has been enough to trigger mutagenicity, but not revivification.  New tests on a wider range of subjects seem to suggest that larger mammals - dogs and monkeys, thus far - are susceptible to both effects.",
```
Comment (tenmillimaster) [inline data/json/snippets/lab.json]:
```suggestion
      "Earlier conjecture that revivification occurred only in humans was premature.  Per earlier experiments, exposure introduces a persistent, low level infection in all mammal subjects, but quickly enters a form of stasis and seems to go dormant.  In most of our experimental subjects this has been enough to trigger mutagenicity, but not revivification.  New tests on a wider range of subjects seem to suggest that larger mammals - dogs and monkeys, thus far - are susceptible to both effects.",
```

File: data/json/snippets/lab.json [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+      "Post-mortem revivification of subject T3D was arrested by an immediate autopsy per XEDRA-40's new protocol: we weren't looking to study zombies today.  The internal changes were significant, with large parts of the subject's internal anatomy replaced with new or duplicated organs.  Dr. Versed over in Entomology is convinced some of his organs are scaled-up derivations of fruit fly anatomy now.",
+      "We were able to isolate what we believe to be re-adapted XE037 from subject T3D's cadaver.  Transdimensionally sourced XE037 is not isolated, but this post-mutagenic, post-revivification stuff seems to obey local physical laws a bit more.  We can pick it up on spectroscopy, but it has comparable physical properties to the sampels of XE037 we obtained \"abroad\".  Samples of the enhanced XE037, labeled PE012, have been sent to several researchers for further study.",
```
Comment (tenmillimaster) [inline data/json/snippets/lab.json]:
```suggestion
      "We were able to isolate what we believe to be re-adapted XE037 from subject T3D's cadaver.  Transdimensionally sourced XE037 is not isolated, but this post-mutagenic, post-revivification stuff seems to obey local physical laws a bit more.  We can pick it up on spectroscopy, but it has comparable physical properties to the samples of XE037 we obtained \"abroad\".  Samples of the enhanced XE037, labeled PE012, have been sent to several researchers for further study.",
```

**Pull request merged**